### PR TITLE
update grafana dashboard

### PR DESCRIPTION
**Pull request by vashian**
## Description
updating grafana dashboard for two new metrics 
`minio_bucket_traffic_sent_bytes`
`minio_bucket_traffic_received_bytes`

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [*] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

**Comment by harshavardhana** [comment]
Please add a screenshot here.

**Comment by vashian** [comment]
Screenshot from 2022-07-21 12-11-34

**Comment by vashian** [comment]
image

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: docs/metrics/prometheus/grafana/minio-dashboard.json [hunk 1]
```diff
@@ -93,8 +42,10 @@
   "liveNow": false,
   "panels": [
     {
-      "cacheTimeout": null,
-      "datasource": "${DS_PROMETHEUS}",
+      "datasource": {
+        "type": "prometheus",
+        "uid": "n5pUBg3nz"
+      },
       "description": "",
```
Comment (harshavardhana) [inline docs/metrics/prometheus/grafana/minio-dashboard.json]: You need to make sure to generate the JSON from the grafana dashboard as "generic" export.

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :x: [more...](#d6aa950f-2001-4ebe-a14e-e699bb64ad2a) |
<a name="d6aa950f-2001-4ebe-a14e-e699bb64ad2a"></a>
### 15357-965ec28/mint-pools.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-k8s8:32382
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0
RUN_ON_FAIL:          0

To get logs, run 'docker cp :/mint/log /tmp/mint-logs'

(1/14) Running awscli tests ... done in 53 seconds
(2/14) Running aws-sdk-go tests ... done in 1 seconds
(3/14) Running aws-sdk-java tests ... done in 1 seconds
(4/14) Running aws-sdk-php tests ... done in 45 seconds
(5/14) Running aws-sdk-ruby tests ... done in 7 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 29 seconds
(8/14) Running minio-go tests ... done in 1 minutes and 17 seconds
(9/14) Running minio-java tests ... FAILED in 1 minutes and 5 seconds
{
  "name": "minio-java",
  "function": "listObjects()",
  "args": "[bucket, prefix, recursive, 1050 objects]",
  "duration": 56747,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = BucketNotEmpty, message = The bucket you tried to delete is not empty, bucketName = minio-java-test-2b7kio2, objectName = null, resource = /minio-java-test-2b7kio2, requestId = 1703EFA2D4580F84, hostId = 57ed50eb-d1e2-42b1-bbef-033bbd0e4b1d)\nrequest={method=DELETE, url=http://minio-k8s8:32382/minio-java-test-2b7kio2, headers=Host: minio-k8s8:32382\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.4.3\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20220721T194250Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20220721/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=409, headers=Accept-Ranges: bytes\nContent-Length: 328\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nServer: MinIO\nStrict-Transport-Security: max-age=31536000; includeSubDomains\nVary: Origin\nVary: Accept-Encoding\nX-Amz-Request-Id: 1703EFA2D4580F84\nX-Content-Type-Options: nosniff\nX-Xss-Protection: 1; mode=block\nDate: Thu, 21 Jul 2022 19:42:50 GMT\n}\n >>> [io.minio.S3Base$1.onResponse(S3Base.java:690), okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519), java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128), java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628), java.base/java.lang.Thread.run(Thread.java:829)]"
}
(9/14) Running minio-js tests ... done in 1 minutes and 6 seconds
(10/14) Running minio-py tests ... done in 1 minutes and 34 seconds
(11/14) Running s3cmd tests ... done in 24 seconds
(12/14) Running s3select tests ... done in 5 seconds
(13/14) Running versioning tests ... done in 3 minutes and 5 seconds

Executed 13 out of 14 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**